### DISCUSSION TITLE
I just want to thank every contributor  in here

### DISCUSSION THREAD

**Discussion by fballan93** [directus/directus#5231] [Archive]
RC-0.60 is the best thing happened to me . the "Relational consistency" fix is the biggest thing to happen for my project . although it is not as visually appealing as the old tables BUT still THANK YOU they function exactly as expected. the display menu  was different  from the interface  and to unify them is a great step. how they are presented now in the interface layout is amazing  . what is your favorite  fix in the latest RC ?.

**Comment by rijkvanzanten** [comment]
Thank you! ❤️ 

My personal favorite thing in rc60 must be the groundwork for supporting update/delete by query 😄

**Comment by benhaynes** [comment]
I was really pushing for the relational consistency update, so I think that's my favorite. That said, I know there's more to be done on that side of things... especially based on all the community feedback we've received. Looking forward to finishing up the "polish" on those interfaces, including:

* Options for toggling Add/Select buttons
* Drag and Drop zone for Multiple Files
* Updated Translation UI

**Reply by fballan93** [reply]
And we all appreciate everyone efforts to push what is efficient for most users.  and I couldn't agree more on all your points you mentioned
